### PR TITLE
[stable/nginx-ingress] Update to 0.10.x release

### PR DESCRIPTION
**Pull request by cap10morgan**
Upgrade default upstream version to 0.10.2 and fix issues that that causes. Supplants #3180 and incorporates changes in #3369 because they are needed here.

### CODE REVIEW

File: stable/nginx-ingress/README.md [hunk 1]
```diff
@@ -46,8 +46,8 @@ The following tables lists the configurable parameters of the nginx-ingress char
 Parameter | Description | Default
 --- | --- | ---
 `controller.name` | name of the controller component | `controller`
-`controller.image.repository` | controller container image repository | `k8s.gcr.io/nginx-ingress-controller`
-`controller.image.tag` | controller container image tag | `0.9.0-beta.15`
+`controller.image.repository` | controller container image repository | `quay.io/kubernetes-ingress-controller/nginx-ingress-controller`
+`controller.image.tag` | controller container image tag | `0.9.0`
```
Comment (unguiculus) [inline stable/nginx-ingress/README.md]: 0.10.2

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by cap10morgan** [comment]
@unguiculus Updated. Thanks for catching that!

**Comment by marc-sensenich** [comment]
@unguiculus Are there any additional limitations for getting this chart merged in? I saw #3418 is open as well, I didn't know if there was a plan to merge that first before releasing this PR so that both a patch and minor version would be released

**Pull request merged**